### ISSUE TITLE
Option to sort books chronologically

### ISSUE THREAD

**Issue by WillAdams** [Librum-Reader/Librum#4]
Ability to sort by publication/chronological/historical order. 

Example use cases/data to sort on: 

 - read biographies of famous persons, in chronological order, by date of birth or death  
 - publication date --- this will make reading complicated series simpler, allowing the recreation of the experience of reading books as they came out
 - date book was originally written
 - for fiction, time period book is set in --- this would allow reading successive works of fiction, say for a given theme or local in chronological order to see how the treatment changed

Basically, allow all the metadata which used to be on library card catalogs (I believe the Library of Congress captures all this)

**Comment by Creapermann** [comment]
@WillAdams, I like the idea of chronological book sorting. The problem here would be, that data like when the author dies, usually isn't provided in the books metadata, thus this information would need to be manually set.

A sort option for the publication date, is something I'll add
